x-pack/plugins/lists/common/schemas/request/delete_list_item_schema.test.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { left } from 'fp-ts/lib/Either';
+import { pipe } from 'fp-ts/lib/pipeable';
+
+import { exactCheck, foldLeftRight, getPaths } from '../../siem_common_deps';
+
+import { DeleteListItemSchema, deleteListItemSchema } from './delete_list_item_schema';
+import { getDeleteListItemSchemaMock } from './delete_list_item_schema.mock';
+
+describe('delete_list_item_schema', () => {
+  test('it should validate a typical exception list item request', () => {
+    const payload = getDeleteListItemSchemaMock();
+    const decoded = deleteListItemSchema.decode(payload);
+    const checked = exactCheck(payload, decoded);
+    const message = pipe(checked, foldLeftRight);
+
+    expect(getPaths(left(message.errors))).toEqual([]);
+    expect(message.schema).toEqual(payload);
+  });
+
+  test('it should NOT accept a value for "namespace_type" since it does not require one', () => {
```

Comment:
Lists don't have `namespace_type` right?

Reply:
Yeah that's true. I think I was just mindless coding and copying things over.